aws/src/utils/safe_object_from_dict.py

```diff
@@ -16,12 +16,12 @@ def safe_object_from_dict(object_type: type[T], data: dict) -> T:
 
     for key in list(data.keys()):
         if key not in object_type.__annotations__.keys():
-            print(f"WARNING: {key} not in {object_type.__name__}")
+            # print(f"WARNING: {key} not in {object_type.__name__}")
             del data[key]
 
     for key in object_type.__annotations__.keys():
         if key not in data.keys():
-            print(f"WARNING: {key} from {object_type.__name__} not in data")
+            # print(f"WARNING: {key} from {object_type.__name__} not in data")
```

Comment:
Remove?

Reply:
Uncommented in 43da1efac99923d3e4451b4473d4d7d0143ec7e6